tests/Organizer/CommandHandler/UpdateWebsiteHandlerTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Organizer\CommandHandler;
+
+use Broadway\CommandHandling\CommandHandler;
+use Broadway\CommandHandling\Testing\CommandHandlerScenarioTestCase;
+use Broadway\EventHandling\EventBus;
+use Broadway\EventStore\EventStore;
+use CultuurNet\UDB3\Language;
+use CultuurNet\UDB3\Organizer\Commands\UpdateWebsite;
+use CultuurNet\UDB3\Organizer\Events\OrganizerCreatedWithUniqueWebsite;
+use CultuurNet\UDB3\Organizer\Events\WebsiteUpdated;
+use CultuurNet\UDB3\Organizer\OrganizerRepository;
+use CultuurNet\UDB3\Title;
+use ValueObjects\Web\Url;
+
+class UpdateWebsiteHandlerTest extends CommandHandlerScenarioTestCase
+{
+    protected function createCommandHandler(EventStore $eventStore, EventBus $eventBus): CommandHandler
+    {
+        return new UpdateWebsiteHandler(new OrganizerRepository($eventStore, $eventBus));
+    }
+
+    /**
+     * @test
+     */
+    public function it_handles_updating_the_website(): void
+    {
+        $id = '5e360b25-fd85-4dac-acf4-0571e0b57dce';
+
+        $organizerCreated = new OrganizerCreatedWithUniqueWebsite(
+            $id,
+            new Language('nl'),
+            Url::fromNative('https://www.madewithlove.be'),
+            new Title('Organizer Title')
+        );
+
+        $this->scenario
+            ->withAggregateId($id)
+            ->given([$organizerCreated])
+            ->when(new UpdateWebsite($id, Url::fromNative('https://www.publiq.be')))
+            ->then([new WebsiteUpdated($id, Url::fromNative('https://www.publiq.be'))]);
```

Comment:
Nit: I usually also add a test in these command handler test files to assert that there is no event recorded when there was no change. I know this overlaps with the tests for the aggregate itself but it's easy to do here, saves us a test in another file, and tests the whole write side a bit more in an "integrated" way. But not blocking or required.

Reply:
That is indeed also an option using "features" tests instead of unit tests.